README.md

```diff
@@ -1,39 +1,48 @@
+[![Coverage Status](https://coveralls.io/repos/github/RipMeApp/ripme/badge.svg?branch=main)](https://coveralls.io/github/RipMeApp/ripme?branch=main)
+
+The current active development repo for RipMe is located at [ripmeapp2/ripme](https://github.com/ripmeapp2/ripme/).
+
+RipMe has been maintained with ♥️ and in our limited free time by the following
+people, roughly in order from most recent primary developer, with current
+activity marked by color of the indicator:
 
-RipMe is maintained with ♥️ and in our limited free time by **[@MetaPrime](https://github.com/metaprime)**, **[@cyian-1756](https://github.com/cyian-1756)** and **[@kevin51jiang](https://github.com/kevin51jiang)**. If you'd like to contribute but aren't good with code, help keep us happy with a small contribution! Chat on [gitter](https://gitter.im/RipMeApp/Lobby).
+- **[@soloturn](https://github.com/soloturn)** 🟢,
+- **[@cyian-1756](https://github.com/cyian-1756)** 🟥,
+- **[@kevin51jiang](https://github.com/kevin51jiang)** 🟥,
+- **[@MetaPrime](https://github.com/metaprime)** 🟡,
+- and its original creator, **[@4pr0n](https://github.com/4pr0n)** 🟥.
 
-[![Tip with PayPal](https://img.shields.io/badge/PayPal-Buy_us...-lightgrey.svg)](https://www.paypal.me/ripmeapp)
-[![Tip with PayPal](https://img.shields.io/badge/coffee-%245-green.svg)](https://www.paypal.com/paypalme/ripmeapp/send?amount=5.00&currencyCode=USD&locale.x=en_US&country.x=US)
-[![Tip with PayPal](https://img.shields.io/badge/beer-%2410-yellow.svg)](https://www.paypal.com/paypalme/ripmeapp/send?amount=10.00&currencyCode=USD&locale.x=en_US&country.x=US)
-[![Tip with PayPal](https://img.shields.io/badge/lunch-%2420-orange.svg)](https://www.paypal.com/paypalme/ripmeapp/send?amount=20.00&currencyCode=USD&locale.x=en_US&country.x=US)
-[![Tip with PayPal](https://img.shields.io/badge/dinner-%2450-red.svg)](https://www.paypal.com/paypalme/ripmeapp/send?amount=50.00&currencyCode=USD&locale.x=en_US&country.x=US)
-[![Tip with PayPal](https://img.shields.io/badge/custom_amount-...-lightgrey.svg)](https://www.paypal.me/ripmeapp)
+If you'd like to become a maintainer, ask an active maintainer to be added to the team.
+
+Chat with the team and community on [gitter](https://gitter.im/RipMeApp/Lobby) and [reddit.com/r/ripme](https://www.reddit.com/r/ripme/)
 
 # About
 
 RipMe is an album ripper for various websites. It is a cross-platform tool that runs on your computer, and
-requires Java 17. RipMe has been tested and confirmed working on Windows, Linux and MacOS.
+requires Java 23 to run. RipMe has been tested and is confirmed working on Windows, Linux, and MacOS.
 
 ![Screenshot](https://i.imgur.com/UCQNjeg.png)
 
 ## Downloads
 
-Download `ripme.jar` from the [latest release](https://github.com/ripmeapp2/ripme/releases). For information about running the `.jar` file, see 
+Download `ripme.jar` from the [latest release](https://github.com/ripmeapp2/ripme/releases). For information about running the `.jar` file, see
```

Comment:
ripme instead of ripme2

Reply:
Well for now latest releases are on ripmeapp2/ripme, unless you're planning to merge development back into ripmeapp/ripme now?